apps/remix-cars-service/app/components/details/item.jsx

```diff
@@ -16,47 +17,41 @@ export default function Item({ item, setItem }) {
 
 	return (
 		<>
-			<div className='row'>
-				<div className="label">
-					<DealIcon/>
-				</div>
+			<div className="row">
+				<ItemImage size="l" />
 
-				<div className="main">
+				<div className="column">
 					<h2 className="cui5-text">
 						{item.title}
 					</h2>
 
-					<Tags items={['New', item.price]}/>
-
-					<ItemStatus status={item.status}/>
-				</div>
-			</div>
 
-			<div className='row'>
-				<div className="label">
-					<CalendarIcon/>
-				</div>
-
-				<div className="main">
-					<div className="label">
-						<span className="font-size--s">Delivery: {item.price}</span>
+					<div className="label font-size--s">
+						<span>Condition:&nbsp;</span>
```

Comment:
Probably `&nbsp;` can be replaced with whitespace?

```suggestion
						<span>Condition:{' '}</span>
```